code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -90,6 +90,252 @@
 			new /obj/item/clothing/suit/ascetic(src)
 		if(31)
 			new /obj/item/kitchen/knife/envy(src)
+		if(32)
+			new /obj/item/sharpener/fire(src)
+
+/obj/structure/closet/crate/necropolis/tendril/greater
+	desc = "It's watching you wearily. It seems terribly bloated."
+
+/obj/structure/closet/crate/necropolis/tendril/greater/PopulateContents()
+	var/loot = rand(1,32)
+	switch(loot)
```

Comment:
having 3 switch statements seems like the worst possible way to do this.

Reply:
it is, and I was planning to ask Marg to walk me through making a loop before the whole VSC debacle happened.

Reply:
i see